File: MyClub/MyClub.Model/Responses/StadiumSectorResponse.cs
using System;

namespace MyClub.Model
{
    public class StadiumSectorResponse
    {
        public int Id { get; set; }
        public int Capacity { get; set; }
        public string Code { get; set; } // A1, A2, B1, B2, B3, etc.
        public string SideName { get; set; }
    }
}

File: MyClub/UI/myclub_mobile/lib/models/responses/stadium_sector_response.dart

class StadiumSectorResponse {
  int id;
  String code;
  String sideName;
  int capacity;

  StadiumSectorResponse({
    required this.id,
    required this.code,
    required this.sideName,
    required this.capacity,
  });

  factory StadiumSectorResponse.fromJson(Map<String, dynamic> json) {
    return StadiumSectorResponse(
      id: json['id'] ?? 0,
      code: json['code'] ?? '',
      sideName: json['sideName'] ?? '',
      capacity: json['capacity'] ?? 0,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'code': code,
      'sideName': sideName,
      'capacity': capacity,
    };
  }
}
